api/pkg/api/app.go

```diff
@@ -85,25 +81,6 @@ func MakeApp(cfg *setup.Configuration) *APIApplication {
 	return apiApp
 }
 
-// Copied from https://gist.github.com/Rican7/39a3dc10c1499384ca91
-// with a slight tweak to make "ID" convert to "id" instead of "i_d"
-func MarshalJSON(val interface{}) ([]byte, error) {
-	var keyMatchRegex = regexp.MustCompile(`\"(\w+)\":`)
-	var wordBarrierRegex = regexp.MustCompile(`(\w{2,})([A-Z])`)
-	marshalled, err := json.Marshal(val)
-
-	converted := keyMatchRegex.ReplaceAllFunc(
-		marshalled,
-		func(match []byte) []byte {
-			return bytes.ToLower(wordBarrierRegex.ReplaceAll(
-				match,
-				[]byte(`${1}_${2}`),
-			))
-		},
-	)
-	return converted, err
-}
```

Comment:
This was causing some unexpected behavior, putting the field in a struct tag works better